internal/cba.go

```diff
@@ -94,6 +110,11 @@ func getTransportConfig(settings *DialSettings) (*transportConfig, error) {
 	if !shouldUseS2A(clientCertSource, settings) {
 		return &defaultTransportConfig, nil
 	}
+	if settings.UniverseDomainNotGDU() {
+		return &transportConfig{
```

Comment:
if you're returning an error, should you build a transport config at all or just return nil?

Reply:
I'll change this to `nil`, along with the early return above it from which I copied this code. Not sure of the reason for it. I was feeling a bit risk-adverse to changing it, but static inspection shows the first return value is not used by callers when err is non-nil. (As you would expect!)

Thanks!